### PR TITLE
Ch/fixnaming

### PR DESCRIPTION
**Pull request by ginihumer**
adjust the naming for tab panels

### CODE REVIEW

File: src/Application.tsx [hunk 1]
```diff
@@ -193,7 +193,7 @@ export const Application = connector(
               >
                 <Tab
                   value={0}
-                  icon={<img src={PSEIcons.Dataset} />}
+                  icon={<SvgIcon style={{ fontSize: 64 }} viewBox="0 0 18.521 18.521" component={PSEIcons.Dataset} />}
```
Comment (dvmoritzschoefl) [inline src/Application.tsx]: Could you create a component for that? fontSize and viewBox do not change only the component, which could be passed as a prop

**Review by dvmoritzschoefl — CHANGES_REQUESTED** [review]
@ginihumer looks good, if the changes to the naming are discussed @puehringer (for cime) and marc

for the svg icon please make a component since its copy pasted 5 times

File: src/Application.tsx (hunk 1 above)
Comment (ginihumer) [reply src/Application.tsx]: done

**Review by puehringer — APPROVED** [review]

**Comment by puehringer** [comment]
@dvmoritzschoefl Feel free to merge this if it works for you.

**Review by dvmoritzschoefl — APPROVED** [review]

**Pull request merged**